src/StoreKeeper/WooCommerce/B2C/Tools/OrderHandler.php

```diff
@@ -118,14 +119,16 @@ protected function isSyncAllowed(int $orderId): bool
         if (StoreKeeperOptions::isOrderSyncEnabled()) {
             $order = new WC_Order($orderId);
             $orderCreatedDate = $order->get_date_created();
-            $orderCreatedDate = date('Y-m-d', strtotime($orderCreatedDate));
-            $orderSyncFromDate = StoreKeeperOptions::get(StoreKeeperOptions::ORDER_SYNC_FROM_DATE);
 
-            if (is_null($orderSyncFromDate)) {
+            if (is_null(StoreKeeperOptions::get(StoreKeeperOptions::ORDER_SYNC_FROM_DATE))) {
                 return true;
             }
 
-            if (strtotime($orderSyncFromDate) <= strtotime($orderCreatedDate)) {
+            $orderSyncFromDate = DatabaseConnection::formatFromDatabaseDate(
+                StoreKeeperOptions::get(StoreKeeperOptions::ORDER_SYNC_FROM_DATE)
+            );
+
+            if (!is_null($orderCreatedDate) && strtotime($orderSyncFromDate->format('Y-m-d')) <= strtotime($orderCreatedDate->format('Y-m-d'))) {
```

Comment:
make sure those have same timezone before comparing (the wp_timezone)

Reply:
Made sure bot are in UTC before comparing.